spec/controllers/api/v1/subject_queues_controller_spec.rb

```diff
@@ -31,18 +34,50 @@
 
   describe "#update" do
     let(:test_attr) { :set_member_subject_ids }
-    let(:test_attr_value) { subjects.map(&:id) }
+    let(:test_attr_value) { subject_ids }
+    let(:test_relation_ids) { subject_ids }
```

Comment:
the api requires subject ids to be passed in which are then converted to sms ids on the fly, so i need different expectations. I'm running bisect to get a reproducible test case on this.